src/CollectorPapi.cc

```diff
@@ -1,54 +1,42 @@
 #include "collector_papi.h"
 #include <papi.h>
-namespace vapro {
 
+#define checkPapi(call,stdpapi)                                                        \
+    {                                                                          \
+        auto err = call;                                                       \
+        if (stdpapi != err) {                                                  \
+            fprintf(stderr, "PAPI error in %s:%i : %s.\n", __FILE__, __LINE__, \
+                    PAPI_strerror(retval));                                    \
+            exit(EXIT_FAILURE);                                                \
+        }                                                                      \
+    }
```

Comment:
This macro is already defined in `include/collector_papi.h`. This overload with two paramters seems to be seldom used as most of the return values are `PAPI_OK`. I suggest using the original `papiCheck` without a custom return value.